app/routes/PairingProcess/components/CreateAccount/pairingNotifications.js

```diff
@@ -0,0 +1,62 @@
+import firebase from 'firebase'
+import EthUtil from 'ethereumjs-util'
+import Web3 from 'web3'
+
+import config from '../../../../../config'
+
+export const setUpNotifications = () => {
+  const messaging = setUpFirebase()
+  return messaging.getToken()
+}
+
+const setUpFirebase = () => {
+  const configFirebase = {
+    authDomain: config.firebase.authDomain,
+    databaseURL: config.firebase.databaseURL,
+    projectId: config.firebase.projectId,
+    storageBucket: config.firebase.storageBucket,
+    messagingSenderId: config.firebase.messagingSenderId
+  }
+  firebase.initializeApp(configFirebase)
+  return firebase.messaging()
+}
+
+export const authPushNotificationService = (pushToken, privateKey) => {
+  const url = config.pushNotificationServiceUrl
+  const headers = {
+    'Accept': 'application/json',
+    'Content-Type': 'application/json',
+  }
+  const body = generateAuthContent(pushToken, privateKey)
+
+  return fetch(url, {
+    method: 'POST',
+    headers,
+    body,
+  })
+    .then((response) => {
```

Comment:
```.then( response => response.status === 201)```
The question is, as far as I remember if you receive other status different than 2XX the ```catch``` will be executed.

Why are you just checking 201? What about the others 20X?

Reply:
The push notification service specifications indicate that a HTTP 201 is returned if OK

Reply:
I think if you just do:
```
return fetch(url, {
    method: 'POST',
    headers,
    body,
 })
```
And in the class where you use this function:
```
try {
    await authPushNotificationService(token, currentAccount.getPrivateKey())    
} catch (err) {
    console.error(err)
}
this.setState({ loading: false })
this.props.onCreateAccount(currentAccount.getChecksumAddressString(), encryptedMnemonic, hmac)
```
You achieve the same in a cleaner way. But this is just my opinion. ;)